contracts/upgradeable_contracts/BasicHomeBridge.sol

```diff
@@ -155,4 +173,41 @@ contract BasicHomeBridge is EternalStorage, Validatable {
 
     function onFailedAffirmation(address, uint256, bytes32) internal {
     }
+
+    function feeManagerContract() public view returns(address) {
```

Comment:
the suggestion is to move all methods related to fee management to a separate contract like `RewardableBridge.sol` and inherit this contract in `HomeBridgeErcToNative`.

Reply:
Done https://github.com/poanetwork/poa-bridge-contracts/pull/132/commits/faf9712db27d7f3a22eda384f2131368791dbfe1